src/transformers/onnx/features.py

```diff
@@ -179,6 +180,16 @@ class FeaturesManager:
             "question-answering",
             onnx_config_cls=DistilBertOnnxConfig,
         ),
+        "flaubert": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "causal-lm",
+            "sequence-classification",
+            # "multiple-choice",
```

Comment:
Ah this is some dead code from previous work on the ONNX API, so please remove it. Although Flaubert has a multiple-choice head, we still need to do some work to provide ONNX support for this general class of models.

Reply:
Should I remove this commented line for all other models ?

Reply:
Thanks for the offer, but no let's do that in a separate PR to keep this one focused on just FlauBERT :)